docs/content/guides/monitoring-and-tracing.md

```diff
@@ -95,25 +91,17 @@ Tracing relies on the trace headers passed through each microservice in an appli
 
 - *At runtime*:
 
-   You can use the [NGINX Service Mesh API]({{< ref "reference/api/api-usage.md" >}}) to update the telemetry configuration.
-
-   For example, send the following payload via `PATCH` to the NGINX Service Mesh API:
-
-   ```json
-   {
-      "op": "replace",
-      "field": {
-         "telemetry": {
-            "exporters": {
-               "otlp": {
-                  "host": "otel-collector.example-namespace.svc",
-                  "port": 4317
-               }
-            },
-            "samplerRatio": 0.1
-         }
-      }
-   }
+   You can use the [NGINX Service Mesh API]({{< ref "api-usage.md#modifying-the-global-mesh-configuration" >}}) to update the telemetry configuration.
+
+   For example, update the `meshconfig` resource with:
```

Comment:
```suggestion
   The example below demonstrates how to update the `meshconfig` resource:
```

Ditto with the last instance, though the fact this is repeated suggests this could probably be rephrased or restructured altogether to cut down on repetition.